client/me/developer/style.scss

```diff
@@ -1,33 +1,80 @@
 @import "@wordpress/base-styles/breakpoints.scss";
+@import "@automattic/typography/styles/variables";
 
-.developer__is_dev_account-fieldset-is-loading {
-	.components-form-toggle {
-		.components-form-toggle__thumb {
-			visibility: hidden;
-		}
+.navigation-header.developer__header {
+	.formatted-header__title {
+		font-family: $brand-serif;
+		font-size: rem(44px);
+		line-height: 52px;
+		letter-spacing: 0.2px;
+		text-align: center;
+		padding-bottom: 8px;
+		word-break: break-word;
+	}
 
-		.components-form-toggle__track {
-			border: none;
-			@include placeholder( --color-neutral-10 );
-		}
+	.formatted-header__subtitle {
+		font-family: "SF Pro Text", $sans;
+		font-size: rem(16px);
+		line-height: 24px;
+		letter-spacing: -0.32px;
+		text-align: center;
 	}
 
-	.components-toggle-control__label {
-		@include placeholder();
-		a {
-			color: inherit;
+	@media ( max-width: $break-medium ) {
+		padding: 40px 32px 24px;
+
+		.formatted-header__title {
+			font-size: rem(32px);
+		}
+
+		.formatted-header__subtitle {
+			display: block;
+			font-size: rem(14px);
 		}
 	}
 }
 
-.developer__settings {
-	padding-bottom: 0;
+.form-fieldset.developer__is_dev_account-fieldset {
+	border-radius: 2px;
+	background: #e5f4ff;
```

Comment:
I will think about it, but it looks totally different on default theme, and the designer used exactly the current color in the design

Reply:
Ah, oops, you're right! It's not really for this purpose, but this might work:

```suggestion
	background: var(--color-link-0);
```

Assuming Design agrees, I do think a colour variable is better here.